### PR TITLE
Docker: change plugin path in custom docker

### LINKED ISSUE #16474: Docs feedback: installation/docker.md recommended volume mount obscures pre-installed plugins
The docker installation first describes how to pre-install plugins (https://grafana.com/docs/installation/docker/#building-a-custom-grafana-image-with-pre-installed-plugins) and then proceeds to recommend a volume for persistent storage (https://grafana.com/docs/installation/docker/#grafana-container-with-persistent-storage-recommended) however, the persistent storage mount obscures the pre-installed plugins and the user is left to wonder why the pre-installed plugins don't show up in the UI.

### PR DESCRIPTION
**Pull request by xlson**
**What this PR does / why we need it**:

This is to make the custom dockerfiles compatible with our instruction
to use persitent volumes for the /var/lib/grafana dir. Without this
change the contents of that folder will be overshadowed by the persitent
volume and none of the pre-installed plugins will be available.

**Which issue(s) this PR fixes**:

Fixes #16474

**Special notes for your reviewer**:

I'm sending this in now before I leave for the office. Thinking that we might want to mention this in the docs and/or readme in the custom folder as well as this differs from the default setup.

### CODE REVIEW

**Review by marefr — APPROVED** [review]
LGTM

File: packaging/docker/custom/ubuntu.Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ARG GRAFANA_VERSION="latest-ubuntu"
+ARG GRAFANA_VERSION="latest"
```
Comment (aknuds1) [inline packaging/docker/custom/ubuntu.Dockerfile]: Why this change?

Comment (marefr) [reply packaging/docker/custom/ubuntu.Dockerfile]: Because of https://github.com/grafana/grafana/pull/21837/files#diff-4dccfdd8f9f661298752023ca816565fR3

Comment (xlson) [reply packaging/docker/custom/ubuntu.Dockerfile]: Bugfix

Comment (aknuds1) [reply packaging/docker/custom/ubuntu.Dockerfile]: Ah

**Review by aknuds1 — APPROVED** [review]
LGTM!
